doc/specs/stdlib_math.md

```diff
@@ -275,3 +275,75 @@ program demo_logspace_rstart_cbase
 
 end program demo_logspace_rstart_cbase
 ```
+## `arange`
+
+### Status
+
+Experimental
+
+### Class
+
+Pure function.
+
+### Description
+
+Create a vector of the `integer/real` type with given fixed spaced values within a given interval.
+
+#### Note
+
+Because of the `i` (`huge(integer :: i) = 2147483647`) index inside the `arange` function , the dimensional maximum length of array created by the `arange` function is `2147483647`. 
+
+### Syntax
+
+`result = [[stdlib_math(module):arange(interface)]](start [, end, by])`
```

Comment:
`by` for the third argument sounds odd to me. I'm trying to think who to say this in words: Arange from start to end by...? It doesn't sound right. 

Why not call this `increment`, or just `incr` if `increment` is too long? Or `step`, like in numpy.arange?

Reply:
OK, `step`.😏